alexxsanya/EPIC-MAIL
# 164728068 bg-challenge-one-feedback

**PR Decription**
This PR is done specifically to improve UX on the system

 [Link to the User Story of the PR](https://www.pivotaltracker.com/story/show/164728068)

**What this PR do?**
- Set bright background for the body element
- Set a better font type
- Resized the logout button to a bigger size
- Removed the time date display html, css & js
- Used clear messages in the UI.
- Set the send & save button below the message body text area
- Removed unused html place holders.

**How to test this PR manually ?** 
- Click [here](https://alexxsanya.github.io/EPIC-MAIL/UI/login.html) to see the changes

@nakimera @bekeplar  @Emmanuelaaron
